maintenance/guess_league_archetypes.py

```diff
@@ -3,8 +3,10 @@
 HOURLY = True
```

Comment:
This code/decision is not part of this PR but shouldn't we be predicting an archetype at INSERT time like we do with Gatherling decks? There's some benefit to not doing it on user time but I'm ok with writes being slow as they are very rare from a user's perspective. We could even fire another process and send the response leaving the other process to complete the prediction work (and we don't care about giving user feedback if it fails). Fine to leave this out of scope :D